infra/helper.py

```diff
@@ -457,15 +457,22 @@ def build_fuzzers_impl(project_name, clean, engine, sanitizer, architecture,
       ['docker', 'run', '--rm', '-i', '--cap-add', 'SYS_PTRACE'] +
       _env_to_docker_args(env))
   if source_path:
-    workdir = _workdir_from_dockerfile(project_name)
-    if workdir == '/src':
-      print('Cannot use local checkout with "WORKDIR /src".', file=sys.stderr)
-      return 1
+    if not mount_location:
```

Comment:
Try not to triple nest conditionals like this.
I think this can be simplified as following:
```if not source_path:
  lines 471-474
elif not mount_location:
   460-469```

Reply:
Done.